splinterd/Cargo.toml

```diff
@@ -38,9 +38,9 @@ clap = "2.32"
 crossbeam-channel = "0.3"
 ctrlc = "3.0"
 cylinder = "0.2.2"
-flexi_logger = "0.14"
 health = { path = "../services/health", optional = true }
 log = "0.4"
+log4rs = "1.0"
```

Comment:
Use "1" here as the minor revision has no meaning. This differentiates when it is added with purpose to specify a minimum, as in the cylinder case.

Reply:
fixed